docs/development/workflow/virtual_pythons.rst

```diff
@@ -164,36 +211,61 @@ addition to activating new ones.
 
       workon ENV
 
-* ` conda`: Activate the environment ``ENV`` with::
+* `conda`_: Activate the environment ``ENV`` with::
 
       conda activate ENV
 
+* `pipenv`_: Activate the environment by ``cd``-ing into the project
```

Comment:
While I realize `cd` is used in the example, it is not the only way to change a directory, and it doesn't work in native Windows shell, so I propose using normal English here.
```suggestion
* `pipenv`_: Activate the environment by changing into the project
```

Reply:
I'm not sure what you mean "it doesn't work in native Windows shell".  `cd` works fine in Windows.  I think it's probably the first DOS command I ever learned :)

Reply:
Fair point though that it's not technically the *only* way to change directories.  I often alias `cd` to `pushd` :)

Reply:
Ops! I must have mixed up `cd` with `ls`. It does work on Powershell now that I check. My bad!

Reply:
It's ok, I think your suggestion was good anyways.